### DISCUSSION TITLE
Changing default status line behavior

### DISCUSSION THREAD

**Discussion by 0xrpheus** [catppuccin/tmux#438] [Help] [answered]
I just installed and configured the theme, for the status-left, I'd like to only have the `0 | <username>` as opposed to `0 | <hostname>` that's being shown right now. 

I managed to display the username using the built-in status, but I'm unable to change the default hostname. 

Here's my .tmux.conf: (I have also attached a screenshot of the status line)
```
unbind r
bind r source-file ~/.tmux.conf

set -g prefix C-\\

set -g mouse on

bind-key h select-pane -L
bind-key j select-pane -D
bind-key k select-pane -U
bind-key l select-pane -R

# Configure the catppuccin plugin
set -g @catppuccin_flavor "mocha"
set -g @catppuccin_window_status_style "basic"

run ~/.config/tmux/plugins/tmux/catppuccin.tmux

# List of plugins
set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'catppuccin/tmux#v2.1.1'

# Make the status line pretty and add some modules
set -g status-right-length 100
set -g status-left-length 100

set -g status-left "#{E:@catppuccin_status_user}"

set -g status-right "#{E:@catppuccin_status_application}"
set -ag status-right "#{E:@catppuccin_status_session}"
set -agF status-right "#[fg=#{@thm_crust},bg=#{@thm_teal}] ##{whoami} "

# Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
run '~/.tmux/plugins/tpm/tpm`
```
![Screenshot 2024-11-21 at 8 33 53 PM](https://github.com/user-attachments/assets/43626e5a-35e1-41b1-a2e3-a57e1f11d0f8)

**Comment by 0xrpheus** [comment, marked as answer]
lol i figured it out, it's not status-left, it's window title :")

**Reply by hywan9** [reply]
I'm having same problem. How did you solve it?
<img width="1582" alt="image" src="https://github.com/user-attachments/assets/697b9099-01bd-40df-ae8e-d02c2c1cdbe7" />

**Reply by kjnsn** [reply]
Did you read https://github.com/catppuccin/tmux/blob/main/docs/guides/troubleshooting.md?

**Reply by 0xrpheus** [reply]
@ke1ewang Try removing line 15 and add the following lines after line 9
```
set -g @catppuccin_window_current_text "#(whoami)"
set -g @catppuccin_window_text "#(whoami)"
```

**Reply by hywan9** [reply]
> Did you read https://github.com/catppuccin/tmux/blob/main/docs/guides/troubleshooting.md?

It's solved. Thank u :)

**Reply by hywan9** [reply]
> It's solved. Thank u :)

No need to delete the line 9. I've solved it. Thank u :)
